pkg/containermanager/container_manager.go

```diff
@@ -16,13 +16,13 @@ type BindConfig struct {
 }
 
 type Binding struct {
-	HostIP string
-	Port   string
+	Port string
```

Comment:
are we explicitly attempting to reserve the object for possible future expansion? seems a bit weird to have a struct with a single content. Better to do a typedef of port to string, and then the map can be of Port to Port instead of string to string.

Reply:
That is true, we don't really need this to be a struct anymore. It wasn't intentional, but you bring up a good point. Do we think we would need to expand this in the future?

Reply:
lets keep it simple. we will expand it in the future if needed. `type Port string` sounds good to me.